spec/cb/responses/application/application_form_spec.rb

```diff
@@ -14,9 +24,9 @@
     it 'raises an exception' do
       response_stub.delete('Results')
 
-      expect{Cb::Responses::ApplicationForm.new(response_stub)}.
-        to raise_error(Cb::ExpectedResponseFieldMissing) do |ex|
-          expect(ex.message).to include 'Results'
+      expect { Cb::Responses::ApplicationForm.new(response_stub) }
+        .to raise_error(Cb::ExpectedResponseFieldMissing) do |ex|
+        expect(ex.message).to include 'Results'
```

Comment:
This block looks weird before and after the rubocop change. I think the `.to ...` part can maybe go up a line?